src/main/java/uk/gov/pay/api/model/links/PaymentWithAllLinks.java

```diff
@@ -84,7 +88,8 @@ public static PaymentWithAllLinks valueOf(ChargeFromResponse paymentConnector,
                                               URI selfLink,
                                               URI paymentEventsUri,
                                               URI paymentCancelUri,
-                                              URI paymentRefundsUri) {
+                                              URI paymentRefundsUri,
+                                              URI captureUri) {
```

Comment:
Should have caught this earlier but it should probably be `paymentsCaptureUri` to match the others